indico/modules/events/abstracts/templates/abstract/public.html

```diff
@@ -241,19 +282,28 @@ <h3 class="abstract-title">
             </span>
         </h3>
         <div class="toolbar thin">
-            {% if management %}
+            {% if abstract.can_judge(session.user) and abstract.state.name == 'submitted' %}
                 <div class="group">
                     {# TODO: scroll to decision buttons #}
                     <a href="#" class="i-button highlight icon-hammer">
                         {% trans %}Judge{% endtrans %}
                     </a>
                 </div>
-            {% elif abstract.event_new.can_manage(session.user) %}
-                <div class="group">
-                    <a href="{{ url_for('.manage_abstract', abstract) }}" class="i-button highlight icon-switchon">
-                        {% trans %}Management view{% endtrans %}
-                    </a>
-                </div>
+            {% endif %}
+            {% if abstract.event_new.can_manage(session.user) %}
```

Comment:
ah... then it might not be needed after all. anyway, i'd keep it for now, it's easy to remove later if we notice it's pointless